src/main/java/ru/skypro/homework/mapper/CommentMapper.java

```diff
@@ -0,0 +1,15 @@
+package ru.skypro.homework.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import ru.skypro.homework.dto.Comment;
+import ru.skypro.homework.entitiy.CommentEntity;
+
+
+@Mapper(componentModel = "spring")
+public interface CommentMapper {
+
+    @Mapping(target = "author", source = "author.id")
+    @Mapping(target = "pk",source = "ad.id")
```

Comment:
здесь id комментария именно
```suggestion
    @Mapping(target = "pk",source = "id")
```

Reply:
где маппинг для authorFirstName?